grupo-gomez-lopez/src/components/sharedButton/constants.js

```diff
@@ -0,0 +1,2 @@
+const buttonStyles = {
```

Comment:
Styles shouldn't be declared in a file named constants

Reply:
Done